### DISCUSSION TITLE
Not launching as Non-Steam game

### DISCUSSION THREAD

**Discussion by adityachopra22** [unknownskl/xbox-xcloud-client#889] [General]
Installed two versions of Greenlight on my steam deck, both launch in desktop mode however R2 is mapped to settings for the program and remapping is not possible in desktop mode so it's unusable. Added as non-steam games to library. Added --fullscreen --no-sandbox --connect=option yet neither version seem to launch. Get steam loading symbol then it autoquits. No idea what the error is.

**Comment by unknownskl** [comment]
I've seen this behavior when you map the appImage directly into your Steam library. Have you tried to add Greenlight to the Steam library by using the start menu and add it from there?

**Reply by adityachopra22** [reply]
Do I do this in gaming mode? I can't find an option to do so in the start menu. Does this also happen in desktop mode?

**Reply by adityachopra22** [reply]
I tried adding once again via desktop mode. Did not work. Am I supposed to do it via gaming mode because I cannot see the option to add a non steam game in this mode.
